docs/services/domain-procedures/which-tlds-does-hypernode-offer.md

```diff
@@ -59,6 +59,13 @@ At Hypernode you can order multiple exotic and new TLD domains. View which exten
 | .UK        | €29.95         | €109.99           | €50.00     |
 | .US        | €19.99         | €109.99           |            |
 
-## How to order
+## How to order a new domain
 
-Go to [this page](how-to-use-domains-in-the-control-panel.md#registering-a-new-domain) on how to order your domains in the Control Panel.
+You can order a new domain very easy via your Control Panel. Please follow the steps in [this article](how-to-use-domains-in-the-control-panel.md#registering-a-new-domain) on how to order your domains in the Control Panel. Or click [this link](https://my.hypernode.com/domain/) to go to the Control Panel directly.
+
+## How to transfer you domain to Hypernode
+
+You can start the transfer of your domain in your Control Panel. Instructions on how to do this can be found in [this article](<>)
```

Comment:
This is a missing link? I guess this needs to be `how-to-use-domains-in-the-control-panel.md#transfer-an-existing-domain-to-hypernode`

Reply:
Jep 😊